packages/polar/sample-project/contracts/examples/schema.rs

```diff
@@ -0,0 +1,20 @@
+use std::env::current_dir;
```

Comment:
what happens if user doesn't have schema in contracts ? is this case handled ?

Reply:
Yes, if schema is not there, constructor will log warnings and parseSchema will throw error